test/unit/decode_fixtures.test.cpp

```diff
@@ -186,9 +186,27 @@ TEST_CASE("Read feature multipolygon")
     CHECK(mpoly[0][0][4].y == 0);
 }
 
-TEST_CASE("Read invalid: missing geometry type field")
+TEST_CASE("Read invalid: missing geometry type field, results in feature collection and no layer")
```

Comment:
oops, I meant this to say "results in **an empty** feature collection and no layer"